bokehjs/src/compiler/linker.ts

```diff
@@ -873,6 +873,7 @@ export async function minify(module: ModuleInfo, source: string, ecma: terser.EC
     ecma,
     format: {
       comments: /^!|copyright|license|\(c\)/i,
+      ascii_only: true,
```

Comment:
An explanation for this would be in order.

Reply:
MathJax includes multiple escaped Unicode characters like [this one](https://github.com/mathjax/MathJax-src/blob/41565a97529c8de57cb170e6a67baf311e61de13/ts/core/MmlTree/OperatorDictionary.ts#L185). And Terser [converts it to UTF-8](https://github.com/terser/terser/issues/410) causing an `unexpected token` error to happen. It's advised to use `ascii_only` so Terser would not convert if they are inside strings or regexp https://terser.org/docs/api-reference.html#mangle-properties-options

Reply:
In this case add a link to the issue on that line, so that we don't have to wonder in future.